docs/examples.md

```diff
@@ -514,6 +513,25 @@ gcloud functions deploy local-nuget \
   --entry-point Google.Cloud.Functions.Examples.LocalNuGetPackageFunction.Function
 ```
 
+## Typed GoogleChatBot
+
+The [GoogleChatbot](../examples/Google.Cloud.Functions.Examples.GoogleChatBot)
```

Comment:
This doesn't really explain the purpose of the example though - it's not like we expect people to use the chat app, but it demonstrates how to integrate with Newtonsoft.Json, right? We should draw attention to that in the text.